iam-login-service/src/main/java/it/infn/mw/iam/api/scim/model/ScimUser.java

```diff
@@ -216,6 +217,11 @@ public Boolean getActive() {
     return active;
   }
 
+  public Boolean getServiceAccount() {
```

Comment:
I think we don't need it here anymore, otherwise we get _service account_ claim also in `ScimUser` object.

Reply:
Thanks. It makes sense.